src/styles/components/button.scss

```diff
@@ -0,0 +1,31 @@
+.button {
+  width: 100%;
+  cursor: pointer;
+  border-radius: 8px;
+  border: 2px solid $mainFontColor;
+  background: $mainFontColor;
+  color: #fff;
+  text-align: center;
+  line-height: 52px;
+  font-size: 16px;
+  font-weight: 700;
+
+  transition: all $effectDuration;
+
+  @include onDesktop {
+    grid-column: 1 / 5;
+  }
+
+  &:hover {
+    border-radius: 8px;
+    border: 2px solid #f0f1f4;
+    background: #f8f8fa;
+    color: $mainFontColor;
+  }
+
+  &:active {
+    border: 2px solid #111;
+    background: #111;
+    color: #fff;
```

Comment:
You have several duplicated inlined colors - consider bringing them to variables.